Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -112,6 +117,8 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
             return "Missing BA Token for PayPal App Switch."
         case .missingPayPalRequest:
             return "The PayPal Request was missing or invalid."
+        case .missingClientMetadataID:
+            return "Missing ClientMetadataID"
```

Comment:
Can we make this a bit more explicit of a message? Maybe something like:
```suggestion
            return "Unable to collect riskCorrelationID during the Edit FI or Error Handling flow."
```